src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -844,6 +848,18 @@ bool fuse_type_to_multiclass_nms(const std::shared_ptr<ov::Node>& node, const pr
     });
 }
 
+bool fuse_type_to_multinomial_v13(const std::shared_ptr<ov::Node>& node, const precisions_map& precisions) {
+    auto multinomial = ov::as_type_ptr<opset13::Multinomial>(node);
+    if (!multinomial) {
+        return false;
+    }
+
+    return update_type(0, node, precisions, [&](const element::Type& type) {
+        multinomial->set_convert_type(ov::element::i32);
+        multinomial->set_output_type(0, type, multinomial->get_output_partial_shape(0));
```

Comment:
It's not required - when this function is finished - revalidate_and_infer_types is called for this node.
```suggestion
```